QMLFile.js

```diff
@@ -127,8 +134,7 @@ QMLFile.prototype.parse = function(data) {
             comment = commentArr.join(" ");
 
             match = QMLFile.unescapeString(match);
-
-            var r = this.API.newResource({
```

Comment:
In order to change this factory method `this.API.newResource` for the string resType, you have to implement the `getResourceTypes` method in the `QmlFileType` plugin to tell it that "string" means `ContextResourceString`. It looks like you already did this, so you just need to call `this.API.newResource` as normal.

Reply:
Yeah..  It should use `this.API.newResource` I'll update it.
If I modified it, it works ok with running real App, Because ContexResource setting work doing under Project's init() stage. https://github.com/iLib-js/loctool/blob/development/lib/Project.js#L143  
But the unit test cases, It directly linked ResourceFactory, https://github.com/iLib-js/loctool/blob/development/lib/ResourceFactory.js#L62
I'm stuck how to properly set ContextResourceString instead ResourceString.

Reply:
You already did that correctly in `QMLFileType.prototype.getResourceTypes()`

Reply:
No, It needs more work to make all unit test pass itself. Please see the latest Travis result)
(Even though module works ok when  running with real an app)